app/models/dfe_subject.rb

```diff
@@ -25,6 +25,10 @@ def has_scholarship_and_bursary?
     has_bursary? && has_scholarship?
   end
 
+  def has_early_career_payments?
```

Comment:
it's worth adding tests for this in `dfe_subject_spec.rb`

Reply:
Done